website/frontend/static/picard.flatpakref

```diff
@@ -0,0 +1,12 @@
+[Flatpak Ref]
+Title=Picard
+Name=org.musicbrainz.Picard
+Branch=stable
+Url=https://flathub.org/repo
+IsRuntime=False
+RuntimeRepo=https://sdk.gnome.org/gnome.flatpakrepo
+GPGKey=mQENBFjpVnsBCACgib1NIXmWfiBCg3dsauP7tsy8YgTU/EZLt1Zl5KGyXflz4RZQC7He6K7c5Oj2zABKT3nnbX9Jtdpj5eobzCHH7p16dz4MNtfWygaKFDcM4p+YBrawwQPAoaCHIDjxt99E4xBqyasrbWI+1rg/ZsisZvmIhdmG1LLBdrqOmrzDeiPWbmXdWooP3OnDOdiWqq8jLSpJJgEWgi89edWEKtAgS0L3bT/nmDrH7M26mXf9wPKMH3ArUaDjDWPWTW9ev4yUCN28I8J4Zq4Z17sAywdnJuf9Ak7Bg95tta4dwDdtr69Aozn2eNaxvR3/AuP0REbLvD2kNw/K13kL1eZunTsrABEBAAG0HUZsYXRodWIgPGZsYXRodWJAZmxhdHBhay5vcmc+iQE5BBMBAgAjBQJY6VZ7AhsDBwsJCAcDAgEGFQgCCQoLBBYCAwECHgECF4AACgkQG7gPiPAVqh4WXwf/XgYD1DKmIZM/TlwykBWbJobqsk/IMuhuMVJsUfRvTTWdrdLx65VQIOYDUUonO8hqzGzo09vX6ALMjnLGAZIVpdzjFxejR9vx5mYyFuIXtYWooups9n8dtOW0JCxeL1YkM4bWWHsz6xiRC78y083nUtv+F3K11pTK7s5VlHPeViz8obDST3sjcDVS34e+BQ/LtzEx10CJC++xvaGood62adOs51YpT4SxOn/bbydlN/vtqoQYYYPcWLFEv+k0J/KGEXiXc7A4txDNADlOUjMhgPwhCOTwntu0in6CyN/j2GpsSHg1CWjapATqrkTInIBkq3tlpwRW43dZNtxTQf4XlbkBDQRY6VZ7AQgA0W0dCNogCNbtE8he6/uG0rB9cvXVfnCuJyxKcT8mixvDpYNYl3uvIxjzHHaimIAyrrU7P/9BqMDMwOECwTpGx4UnvjIzSh54hWJHqyJXBXANDn5/4wtsfjMhWYYn4t8pCFz3ErxQYNavNMlOvhvrlSMC9C1rdDqEvizJy+tg1PWl8eOpX/L2shvWLs57GDH6xkGyUXJ+Mw/Ujivs5pMACfOp3pJAr5jSrmqmhboM6+UYP3eJHLDHscn3u3gUJ7vLReUooURChqbkvSGicrDQiNpaPbLULwxjwXkPTKXa6btztMV4GChsZ5YSH2jodbGyw1dSJ77RCYl4kj+3cI9yZwARAQABiQEfBBgBAgAJBQJY6VZ7AhsMAAoJEBu4D4jwFaoePlMIAJqgnKWNYsuP2RTuqoVlcQ/05833Z6NT+/8UYGolLfj6JhOfESgZoDaoSnXcU8Z1D3oVRsJI4ULbU4WBca1Td3D63wIkvT5tuNclNYgfkeexVjSA7f7oFsQQX1s6+Dp5LaOy5X5rGvz0Lcvy8dz7KQYtQpB+40THp9ABye4dsmPa3cRJYgfdxUr/BJo0/a6Prj8qtapQJ3zEXrtR8ya6pB80b0pviE2D3E/Fq9A7TE2v5SGjBWv5NBQNLLVnZvBauNzt12AnZTsFTuMrx3R/AHO6IODM9s6VPlOeLvaCFtWdOrcVAZZO9D1Lqa6+DwwV+DiOZbNVwvMhD/WED3W4nHw=
+
+Comment=Picard
+Description=MusicBrainz's music tagger
+Icon=https://picard.musicbrainz.org/static/img/picard-icon-large.svg
```

Comment:
> Well, I'm sure the file on Flatpak's server still references an icon somewhere

It does not.

There are two concepts here:

* a `.flatpakref` file contains the metadata required to install an application (its name, the repo to take it from, an icon, etc...)
* a `.flatpakrepo` file contains the metadata required to enable a repository, which itself will contain apps

In this thread we're commenting on the `.flatpakref` file, which indeed points to the icon somewhere on the web.

The new version I just pushed has a `.flatpakrepo` file, which only enables the Flathub repo. With this, the icon is not on the web any more, it's inside the Flatpak "package" for Picard.